src/ui/components/Account/AccountListItem.tsx

```diff
@@ -5,12 +5,16 @@ import type {Account as SubstrateChainAccount} from 'src/api/hooks/useAccount';
 import globalStyles from '@ui/styles';
 import Identicon from '@polkadot/reactnative-identicon';
 import {Account} from './Account';
+import {NavigationProp, useNavigation} from '@react-navigation/native';
+import {AppStackParamList} from '@ui/navigation/navigation';
+import {memberDetailsScreen} from '@ui/navigation/routeKeys';
 
 type Props = {
   account: SubstrateChainAccount;
 };
 
 export function AccountListItem({account}: Props) {
+  const navigation = useNavigation<NavigationProp<AppStackParamList>>();
```

Comment:
With this approach every item in the list is rendering the `useNavigation` hook which is not ideal. You can move it to the parent and pass the callback to the item component.